### PR TITLE
[Mining] Recent mining hash speed

### PR DESCRIPTION
**Pull request by Zannick**
### Issue

In `getmininginfo`, it isn't clear that it computes the `hashspeed` by counting all hashes made since the start of hashing and dividing by the elapsed time, which tends to smooth out fluctuations but also (prior to #966) can include periods of time when the miner is off or the number of threads different.

### Solution

This adds a field to `getmininginfo` called `hashspeed_recent` which is a sum of each thread's most recent round of hashing (where a round is either a fixed number of hashes or ends with a new block).

### Tested

On regtest chain with sha256d and randomx.

### CODE REVIEW

**Comment by Curay1954** [comment]
Scam complaint received from slice wallet SbJsmsH2DF5cQsTNrfWHyVvKQrJ7NqRrXc

Funds were removed from this wallet on July 29 to another wallet

Shjn1gLPKF6n1Et9uFFmsorx4x5Sdi8qJH

On July 29 as well,these funds were supposed to be
Deposited to clients Bank of America
Don't understand why funds were moved to another wallet
Instead of Bank of America has to reply to scam complaint

**Review by CALISOULB — APPROVED** [review]

**Review by CaveSpectre11 — APPROVED** [review]
utACK https://github.com/Veil-Project/veil/pull/967/commits/79102e2d2b75d7b8746d1281e6af2cc0b4c8876a

**Comment by Curay1954** [comment]
Hi, I am confused, mining was done and we were waiting for review and approval

Can you give me an update? Are you going to be able to send my Sumcoin back to my wallet to be able to make a bank deposit?

Please let me know as soon as you can if possible
Thanks

Curay54
Raymond Bailly

**Comment by seanPhill** [comment]
> Hi, I am confused, mining was done and we were waiting for review and approval Can you give me an update? Are you going to be able to send my Sumcoin back to my wallet to be able to make a bank deposit? Please let me know as soon as you can if possible Thanks Curay54 Raymond Bailly

What is "Sumcoin"? Also, the other post you did appeared to be referring to a completely different wallet, currency, and project. ??

**Comment by Curay1954** [comment]
Hi, everything I sent you is related to the same

 Issue, the main problem it to find a way to return my profits from trading with cryptotraders360 this year to my actual Sumcoin Slice wallet.
I did some systems reviewed before sending this email to verify that what I am providing you are true statements.
I am pointed to July 29 as the Origen date because that is the date Cryptotraders360, my trading company sent my profits 16.53 bitcoins to a Sumcoin via Fidelity.

Today I verified the block 2092660 details and it was confirmed July 29 as well as transaction information for slice wallet SbJsmsH2DF5cQsTNrfWHyVvKQrJ7NqRrXc, total 921.002524

Also, same day July 29, profit were sent to Sumcoin slice wallet Shjn1gLPKF6n1Et9uFFmsorx4x5Sdi8qJH, when I reviewed sumexplorer.com I got this
That wallet was not found, the block and transaction were verified.

All these information went thru mining and approved as well as the pull report, block and transactions. The question is, if we have all the proof and information that the funds were
Never given to me or deposit to my bank account, how long do I have to continue waiting for my funds?
The trading manager that worked with me before told me to give him $6000 and he
Will fix it for me, I refused to pay somebody that money, it is like a robbery. I besides that, I already paid Sumcoin fees $45600

Please let me know if you need any other information but I would like to have my funds transferred to my wallet before month end.

Sincerely

Raymond Bailly

**Comment by CaveSpectre11** [comment]
Nothing you are talking about is related to this PR, or even Veil or this project.  You need to go find the organization you are dealing with.  It is not us.

**Comment by WetOne** [comment]
utACK 79102e2

**Review by WetOne — APPROVED** [review]

**Comment by Curay1954** [comment]
Mr, Veil thre we’re intention to bother you, accepts my apology 

Thanks

Ray

**Comment by Curay1954** [comment]
Hi,

I am trying to understand and get a resolution to this case, the facts in this case are not going to change, the main issue here and nobody provided a definite response is:: when I am going to receive my funds??

They are secondary issues that somebody should look at them, like why was the money sent to another wallet that doesn’t exist? What was the purpose? Another one that I found disruptive is the fact the Sumcoin asked me to pay $45600 before the funds were release, but the email sent by the BOSS said, Sumcoin doesn’t charge clients huge amounts but relay in donations , 45k is a lot of money.

I need to know if somebody is going to help me or if you can’t, get a contact to get this done. I have 800k someplace at Sumcoin since July 29 2021, but I need that money in my bank and my trading 

Please respond, thank you

Raymond Bailly

**Comment by Curay1954** [comment]
Mr veil 
I have not bad intentions with my
Comments, I apology if
I offended you

**Pull request merged**
